City-of-Helsinki/drupal-helfi-strategia
# UHF-2093 Update helper

The Update helper upgrade is mainly a change in development practices. That being said, you'll just need to test that the site gets up and running without any significant/fatal errors.

**Notice** This PR will have failing tests until HELfi Platform config module PR is merged, new release has been created and the composer dependencies have been updated to this branch.

How to test:
- Checkout this branch and get the updated version of HELfi Platform config by running `composer require drupal/helfi_platform_config:dev-UHF-2093_update_helper`
- Choose a scenario which suits your current setup:
   - To perform clean install (site is not running and there's no database dump), run `make new`
   - To perform an install to site with existing database dump (dump.sql exists), run `make fresh`
   - To perform an install to existing site (site is running and there's data in the database), run `make drush-cim drush-cr drush-updb drush-uli`
- Check that the site installs without fatal errors and works as before.
